src/pages/events/index.svelte

```diff
@@ -1,90 +1,164 @@
 <script>
-import EventFilters from 'components/EventFilters.svelte'
-import FilteredDisplay from 'components/FilteredDisplay.svelte'
-import { isParticipant, populateEventFilterFrom } from 'data/eventFiltering.js'
-import { join, events, loading } from 'data/events.js'
-import { me } from 'data/user.js'
-
-import Icon from 'fa-svelte'
-import { faExternalLinkAlt, faEdit } from '@fortawesome/free-solid-svg-icons'
-import { goto, params } from '@roxi/routify'
-import { flip } from 'svelte/animate';
-import { fade } from 'svelte/transition';
-
-const format = date => new Date(date).toLocaleDateString([], {
-  month: 'short',
-  day: 'numeric',
-  year: 'numeric',
-  timeZone: 'UTC',
-})
-const logoUrl = event => event.image_file?.url || ''
-
-let eventFilter = {}
-
-$: eventFilter = populateEventFilterFrom($params, $me)
+  import EventFilters from "components/EventFilters.svelte";
+  import FilteredDisplay from "components/FilteredDisplay.svelte";
+  import {
```

Comment:
This import looks short enough that it shouldn't need to be broken on several lines.